src/atoms/InputLabel/inputlabel.scss

```diff
@@ -0,0 +1,23 @@
+@import "../../styles/variables.scss";
+
+.label {
+  font-family: Mulish;
```

Comment:
I'd like to use sass variables for this so if we don't have a variable for `Mulish` yet let's call it `$ui-font: Mulish, sans-serif`